app/src/components/market/common/display_resolution/index.tsx

```diff
@@ -0,0 +1,74 @@
+import moment from 'moment-timezone'
+import React, { DOMAttributes } from 'react'
+import ReactTooltip from 'react-tooltip'
+import styled from 'styled-components'
+
+import { formatDate } from '../../../../util/tools'
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0px;
```

Comment:
```suggestion
  margin: 0px;
```

Maybe use unitless zero.